apps/atlas/src/components/web3mq/LordsChatProvider.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { Client } from 'web3-mq';
+import type { KeyPairsType, EventTypes } from 'web3-mq';
+import { useChatContext } from 'web3-mq-react';
+
+export const getShortAddress = (address = '', num = 5, endNum = 4) => {
+  const strLength = address.length;
+  return (
+    address.substring(0, num) +
+    '...' +
+    address.substring(strLength - endNum, strLength)
+  );
+};
+
+export const botMainKeys = {
+  mainPrivateKey:
+    'sPDsP3D3upCELZj09bDlfLTdQ7TQbv0UjkX0/TvVBvRwtAiyPRa3zjJqjO7XHyIfb424yH0v65tq+FCiVb+VWN7hUmVk7pCNsF91cb8zDCo=',
```

Comment:
are these generic keys? Do we need to use different in production?

Reply:
These keys would definitely need to be put in a safe place during production, or we'd have to find a different way to do non-chat narratives and information display within these groups